enterprise/jakarta.transformer/src/org/netbeans/modules/fish/payara/jakarta/transformer/TransformerConstants.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.fish.payara.jakarta.transformer;
+
+/**
+ *
+ * @author alfon
+ */
+public class TransformerConstants {
+       
+    public static final String PAYARA_TRANSFORMER_PLUGIN = 
+            "fish.payara.transformer:fish.payara.transformer.maven:0.2.12:run";       
```

Comment:
consider upgrading the version from the hard coded default using the repo index. Otherwise it is likely that we will forget about this line until it breaks.

the nbm wizard is in a similar situation and upgrades hardcoded archetype versions by doing this:
https://github.com/apache/netbeans/blob/a56df8c75784ea61c41c5e54f9d0dfd6d96a0ac5/apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/NbmWizardIterator.java#L122-L137

Reply:
@mbien I already improved the way to get last version of the transformer plugin

Reply:
@mbien Can you please help me to try again the tests?